app/scripts/filters/resources.js

```diff
@@ -1084,7 +1084,18 @@ angular.module('openshiftConsole')
       return lastFinishTime;
     };
   })
+  // gets the first status condition.
   .filter('statusCondition', function() {
```

Comment:
`statusConditionIs` filter is the original code, so I aliased `statusCondition` back to it & updated my calls.